web/templates/blocks/sap_instance.html.tmpl

```diff
@@ -28,8 +28,11 @@
                     <td>{{ .Elapsedtime }}</td>
                     <td>{{ .Pid }}</td>
                     <td>
-                        {{- $Textstatus := .Textstatus }}
-                        <span class='badge badge-pill badge-{{ if eq $Textstatus "Running" }}primary{{ else }}danger{{ end }}'>{{ $Textstatus }}</span>
+                        {{- $Dispstatus := .Dispstatus }}
+                        {{ .Dispstatus }}
+                        <span class='badge badge-pill badge-{{ if eq $Dispstatus "SAPControl-GREEN" }}primary{{ else if eq $Dispstatus "SAPControl-YELLOW" }}warning{{ else if eq $Dispstatus "SAPControl-RED" }}danger{{ else if eq $Dispstatus "SAPControl-GRAY" }}secondary{{ end }}'>
+                            {{ $Dispstatus }}
```

Comment:
I don't think we should print `$Dispstatus`, this is an internal reference. I still think that we should print the `$Textstatus`, which is the user friendly output (and as @stefanotorresi commented in the issue, the Dispstatus is a wrapper code to know the state, but each GREEN state can have different user outpus)